webapp/components/experiments/JsonViewer.js

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from 'react';
+import ReactJson from 'react-json-view';
+import { getParameters } from 'root/api/experiments';
+import Breadcrumbs from '@material-ui/core/Breadcrumbs';
+import Typography from '@material-ui/core/Typography';
+import { withStyles } from '@material-ui/core/styles';
+import Button from '@material-ui/core/Button';
+import NavigateNextIcon from '@material-ui/icons/NavigateNext';
+import {
+  fontColor, experimentLabelColor, bgDarker, bgDarkest, primaryColor,
+} from '../../theme';
+
+const useStyles = (theme) => ({
+  root: {
+    width: 'auto !important',
+    flexDirection: 'column',
+    backgroundColor: bgDarker,
+    padding: theme.spacing(2),
+    borderRadius: '0.25rem',
+    "& .MuiBreadcrumbs-separator": {
```

Comment:
Watch out for eslint errors, in this case single-quote instead of double

Context: I refined the rules and fixed all open errors, you will still see many warnings, but we shouldn't have any errors anymore.